nextcloud/Dockerfile.23.0

```diff
@@ -0,0 +1,83 @@
+FROM hoellen/nginx-php:7.4
+
+ARG NEXTCLOUD_VERSION=22.0.0
```

Comment:
Shouldn't this be 23.0.0?

Reply:
Yes ... slipped somehow through while committing it.